tool-openssl/x509_test.cc

```diff
@@ -0,0 +1,111 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#include "openssl/x509.h"
+#include <openssl/err.h>
+#include <gtest/gtest.h>
+#include <stdio.h>
+#include <string>
+#include <vector>
+#include <memory>
+#include "../tool/internal.h"
+#include "internal.h"
+
+X509* CreateAndSignX509Certificate() {
+  X509 *x509 = X509_new();
+  if (!x509) return nullptr;
+
+  // Set validity period
+  if (!X509_gmtime_adj(X509_getm_notBefore(x509), 0) ||
+      !X509_gmtime_adj(X509_getm_notAfter(x509), 31536000L)) {
```

Comment:
Just curious, what does `31536000L` stand for?

Reply:
31_536_000 is the number of seconds in 365 days -- so it's just adding a year?  A comment here would help clarify.